css/bottom-controls.less

```diff
@@ -17,48 +17,43 @@
   }
 
   .animation-button, .settings-icon, .fullscreen-icon {
-    margin: 0 0.2em;
+    margin: 0.5em 0.5em;
+    font-size: 1.5em;
+    height: 1.5em;
+    width: 1.5em;
+    line-height: 1.5em;
     display: inline-block;
-    font-size: 3.5em;
+    text-align:center;
     cursor: pointer;
     color: #777;
+    background-color: #eeeeee;
+    background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #fbfbfb), color-stop(100%, #bcbec0));
+    background-image: -webkit-linear-gradient(top, #fbfbfb, #bcbec0);
+    background-image: -moz-linear-gradient(top, #fbfbfb, #bcbec0);
+    background-image: -ms-linear-gradient(top, #fbfbfb, #bcbec0);
+    background-image: -o-linear-gradient(top, #fbfbfb, #bcbec0);
+    background-image: linear-gradient(top, #fbfbfb, #bcbec0);
```

Comment:
This is really helpful - I figured we had something like this somewhere but didn't spot the link. Will rework accordingly, thanks.